pages/common/script.md

```diff
@@ -0,0 +1,19 @@
+# script
+
+> Make typescript of terminal session.
+
+- Start recording in new file:
```

Comment:
We should start with the simplest option first - `script`. And mention that without any arguments it saves in a file named "typescript"